src/hclib-instrument.c

```diff
@@ -4,13 +4,30 @@
 
 #include <signal.h>
 #include <errno.h>
-#include <aio.h>
 #include <sys/stat.h>
 #include <sys/types.h>
 
 #include "hclib.h"
 #include "hclib-instrument.h"
 
+#ifdef HAVE_AIO_H
```

Comment:
@srirajpaul I'm not a autoconf expert. Am I correct in assuming that HAVE_AIO_H will be set so long as autoconf was able to find aio.h during configuration? So on most systems, it would be set?

Reply:
That is correct. HAVE_AIO_H gets set as  "#define HAVE_AIO_H 1" in hclib-install/include/hclib_config.h if autoconf is able find aio.h in the target platform. In most systems it will be set.